### PR TITLE
suggestion to add updated PR and new issue templates

### PR DESCRIPTION
**Pull request by stuttskl**
## Motivation

I wanted to contribute to this OS project in particular because I believe in the mission and goal of the project. However, I felt that many of the current open issues are beyond my skillset at this time. Despite this, I believe that great documentation and documentation standards can really add value to projects, even in small ways. New issue and PR templates can be a great way to create a standard for reporting new issues, to ensure all necessary details about the bug/how to replicate and provided at the time of opening.

## Implementation

After researching various PR and issue templates, I compiled features from various templates online to bring together the most important details necessary when opening a new issue or PR. Both are in .md files, and are in the .github directory 

## Screenshots/Links

N/A

### CODE REVIEW

**Comment by stuttskl** [comment]
@aviskarkc10 thanks for your comment! Would you like me to instead add items to the existing PR template and re-commit? 
Thank you -- I will check out the Slack channel for this project.

**Comment by aviskarkc10** [comment]
yeah it'd be better to add it in the existing PR template

**Comment by stuttskl** [comment]
Sounds great. I added items to the existing PR template

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## Files
+
+> Files pertinent to the bug or issue you are reporting
+
+## Steps to Reproduce the Problem
+
+> Please list the steps required to attempt to replicate the bug
+
+## Specifications
```
Comment (gitjul) [inline .github/ISSUE_TEMPLATE.md:21]: I'd just add new lines under headings in places where there aren't any for consistency :) e.g. here

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -2,10 +2,23 @@
 
 --> Replace this with text explaining what this change is and why it was made.
 
+## Description
```
Comment (gitjul) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: also here

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -2,10 +2,23 @@
...
+--> Please include a summary of the change and which issue is fixed. 
+
+Fixes # (issue)
+
 ## Implementation
 
 --> Replace this with the main points explaining how the change was made.
 
+## Type of change
```
Comment (gitjul) [inline .github/PULL_REQUEST_TEMPLATE.md:14]: and here

Comment (stuttskl) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: thank you! just added those spaces and pushed the changes : )

**Comment by gitjul** [comment]
@stuttskl oh, also, we make PRs with `integration/webapp` as the base branch. if it's not synced with master, we sometimes need to do that so that the diff is proper. then @sethuraj or @aviskarkc10 take care of merging it to `production/release`, and then to `master`. more about the workflow here: https://docs.google.com/document/d/1zdn3pdDTFzjnnGOGg32x-XRhAE6LET0nhB02RPfIg-Q/edit

**Comment by aviskarkc10** [comment]
Thanks for the contribution @stuttskl and thanks for reviewing the changes @gitjul

Merging this straight into `master` because these changes won't reflect unless they've been merged into the main branch.

**Pull request merged**